.circleci/config.yml

```diff
@@ -322,7 +322,6 @@ workflows:
   weekly-workflow:
     when: << pipeline.parameters.run-weekly-workflow >>
     jobs:
-      - build_and_test_scream
```

Comment:
My preference would be for deleting the job definition for `build_and_test_scream` as well.